recipes/physical_validation/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "physical_validation" %}
+{% set version = "1.0rc4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/shirtsgroup/physical_validation/archive/{{ version }}.tar.gz
```

Comment:
I believe the url should be:

```
https://github.com/shirtsgroup/physical_validation/archive/v{{ version }}.tar.gz
```

note the missing `v` before the version templating.